src/script/components/MessagesList/Message/ContentMessage/MessageFooterLike.tsx

```diff
@@ -44,16 +44,16 @@ const MessageFooterLike: React.FC<MessageFooterLikeProps> = ({message, is1to1Con
       <div className="message-footer-icon">
         <MessageLike className="like-button" message={message} onLike={onLike} />
       </div>
-      <div
-        className={cx('message-footer-label', {
+      <button
+        className={cx('button-reset-default', 'message-footer-label', {
```

Comment:
can we add button type? I think default is submit.

Reply:
have you tested with screen reader? does it announce the name inside span? is the name of the button clear to the user? I think its better to add a aria-describeby in button element.

Reply:
Sure, you are right, it defaults to submit. We should use `button` type as they are not submitting anything :D

Reply:
Yeah, I didn't test with screen readers, I will do it rn and add aria attribute :D

Reply:
Ok, I added button type and put data-uie-name back to close warning button ;)